packages/dotcom-ui-header/src/components/dropdown-navigation/dropdownNavigation.js

```diff
@@ -144,6 +144,57 @@ const trackDropdownView = (options) => {
   elementsToTrack.forEach((el) => observer.observe(el))
 }
 
+/**
+ * Dispatches a custom event with tracking data for Amplitude experiments.
+ *
+ * Remove all relevant code when the experiment is complete.
+ */
+const trackDropdownExposure = () => {
+  const flagDataProNavigation =
+    document.querySelector('[data-flag-pro-navigation]')?.dataset.flagProNavigation ||
+    (document.querySelector('#page-kit-app-context') &&
+      JSON.parse(document.querySelector('#page-kit-app-context').innerText).abTestState?.match(
+        /pro-navigation:([^,]+)/
+      )?.[1])
+
+  if (flagDataProNavigation !== undefined && flagDataProNavigation !== 'no-experiment') {
```

Comment:
```suggestion
  if (flagDataProNavigation && flagDataProNavigation !== 'no-experiment') {
```
Also, curiously, what is the `no-experiment` value? Is it a variant?

Reply:
This code will be removed after the experiment right? Since there will be no need for exposure event afterwards?

Reply:
That's the plan. 
The pro-navigation flag variable (proNavigationVariant) will be removed; only the dropdown toggler prop (showProNavigation) will remain. 
No further next-* PRs should be necessary, ideally.